spec/fixtures/files/defendant_summary/required_fields.json

```diff
@@ -0,0 +1,34 @@
+{
+  "defendantId": "b760daba-0d38-4bae-ad57-fbfd8419aefe",
+  "defendantName": "Bob Smith",
+  "offenceSummary": [
+    {
+      "offenceId": "9aca847f-da4e-444b-8f5a-2ce7d776ab75",
+      "offenceCode": "TH68026C",
+      "offenceTitle": "Conspire to commit a burglary dwelling with    intent to steal",
+      "offenceLegislation": "Contrary to section 1(1) of the    Criminal Law Act 1977.",
+      "proceedingsConcluded": false,
```

Comment:
Are you happy to have all the non-required fields in this? Offence summary only needs Id, code, title